assets/js/googlesitekit/components-gm2/Checkbox.js

```diff
@@ -45,6 +45,7 @@ export default function Checkbox( props ) {
 		onKeyDown,
 		loading,
 		alignLeft,
+		label,
```

Comment:
Added an optional `label` prop that allows considering the `children` contents as additional description.

Reply:
Hmm, I think this is a little confusing as the `label` prop changes the way `children` is handled and it's not very clear in terms of the interface (i.e. props) of the component.

How about we add a `description` prop instead here, so `children` is always handled as a label? That would be a bit easier to reason about.

Maybe a future update to `Checkbox` could be to add a `label` prop and replace all usage of `children` with it, so we end up with `label` and `description` props and no use of `children`, this would further improve the consistency of the interface.

An addition note on this topic - please can you add story variants for this and the GM3 Checkbox that make use of the new `decription` prop (or whatever we end up with)?

Reply:
Added a `description` prop including story variants for it. Thanks!